crates/biome_js_formatter/Cargo.toml

```diff
@@ -29,6 +29,8 @@ smallvec                     = { workspace = true }
 unicode-width                = { workspace = true }
 
 [dev-dependencies]
+biome_css_formatter  = { path = "../biome_css_formatter" }
+biome_css_parser     = { path = "../biome_css_parser" }
```

Comment:
I'm wondering if it could be a circular dependency problem at some point.

Reply:
Actually, I dont know. I'm not familiar with `cargo`. Does a dev dependency participate in the build phase? If there is a cycle, will `cargo` report it?

Reply:
> Cycles aren't allowed, and rust won't compile until it's resolved.
Thank you! TIL

> The first option is maybe the best one
Agree

Reply:
The more I think about it, the more I am inclined to believe that it may be okay to have CSS dependencies in the dev dependencies for tests? It's simpler and prevents the JS formatter tests from being spread across the project. If we move it to another crate, how will we collect statistics about Prettier compatibility?